src/shogun/mathematics/linalg/internal/implementation/Add.h

```diff
@@ -110,7 +111,7 @@ struct add<Backend::NATIVE, Matrix>
 	 * @param beta constant to be multiplied by the second vector
 	 */
 	static void compute(SGVector<T> A, SGVector<T> B, SGVector<T> C,
-		T alpha, T beta)
+		T alpha=1, T beta=1)
 	{
 		REQUIRE(A.vlen == B.vlen, "Vectors should have same length!\n");
```

Comment:
I know its off topic for the patch, but still. We should keep those clean, to avoid user frustration